tests/test_console.py

```diff
@@ -81,18 +78,23 @@ def submit_test_responses():
             db.session.commit()
 
 
+@testing.postgresql.skipIfNotInstalled
 class TestAuthentication(unittest.TestCase):
 
     def setUp(self):
-        self.postgres = Postgresql()
-        Postgresql.clear_cache()
         self.app = server.app.test_client()
         self.app.testing = True
         self.render_templates = False
 
+        with app.app_context():
+            db.create_all()
+            create_initial_users()
+            db.session.commit()
+
     def tearDown(self):
-        Postgresql.clear_cache()
-        self.postgres.stop()
+        with app.app_context():
+            db.session.remove()
+            db.drop_all()
```

Comment:
Somewhere in here we need to clear the cache. It can be done after testing in the tearDown method as per https://pypi.python.org/pypi/testing.postgresql